Notes/Design/GUI.md

```diff
@@ -0,0 +1,23 @@
+* select button to open radial submenue, do it again for the next submenu
+* only on menu per level is open at the same time
+* you can have either painting or modeling tools <- (New image -> painting state -> modeling state -> finish)
+* movement direction and intensity it done with mouse input in normal mode
+
+* every state
+	* color picker
+		* different colors (<- select to change?)
+	* redo
+		* new canvas
+		* redo step
+		* back end of painting state
```

Comment:
What do you mean by back end of painting state?

Reply:
_Back to end of painting state_: the canvas state, before the first rake will be applied

Reply:
good idea; did not thought of this one